src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/publicApi/PublicLicencePolicyServiceIntegrationTest.kt

```diff
@@ -0,0 +1,140 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration.publicApi
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.groups.Tuple
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licencePolicy.LicencePolicy
+import java.nio.charset.StandardCharsets
+
+class PublicLicencePolicyServiceIntegrationTest : IntegrationTestBase() {
+
```

Comment:
Unsure whether to use the 'json' matching version of the test or break down the response into our condition components and check the structure or both?

Reply:
the json version is good, can remove the other version

Reply:
Done